dhis2-android-app/src/main/java/org/dhis2/mobile/utils/TextFileUtils.java

```diff
@@ -37,18 +37,30 @@
 import java.io.FileOutputStream;
 import java.io.FileReader;
 import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 
 public final class TextFileUtils {
     private static final String TAG = TextFileUtils.class.getSimpleName();
 
+    public static void saveLogMessage(Context context, String message) {
```

Comment:
@idelcano this method not use existed standards for this file, can you refactor to for create a overload for WriteTextFile, for example:
writeTextFile(Context context, Directory dir, FileNames name, String data, includeDate)